crates/iota-core/src/transaction_orchestrator.rs

```diff
@@ -360,6 +359,7 @@ where
                     Ok(unfinished_quorum_driver_task.await)
                 }
             };
+            #[expect(clippy::let_and_return)]
```

Comment:
Personal preference or...?

Reply:
No, the expect actually does not work if it's not at the lint site